code/modules/mob/living/brain/brain_item.dm

```diff
@@ -147,6 +147,23 @@
 	icon_state = "brain-x"
 	origin_tech = "biotech=6"
 
+/obj/item/organ/brain/cybernetic/vox
+	name = "Vox cortical stack"
+	slot = "brain"
+	desc = "A vox cortical stack - considered to be their brain. Its existance is the center of debate."
+	zone = "head"
+	icon_state = "voxbrain"
+	status = ORGAN_ROBOTIC
+	origin_tech = "biotech=3"
+
+/obj/item/organ/brain/cybernetic/vox/emp_act(severity)
+	to_chat(owner, "<span class='warning'>Your cortical stack has been damaged!</span>")
```

Comment:
Make it more vague. "Your head hurts...", for example. Bonus points if the message is the same thing as something completely different somewhere in the code.

Reply:
oof ouch my cortical stack